tests/introspection/ApiSignatureTest.cs

```diff
@@ -669,6 +669,8 @@ protected virtual bool Check (char encodedType, Type type)
 				case "System.Byte":
 				// GLKBaseEffect 'instance Boolean get_ColorMaterialEnabled()' selector: colorMaterialEnabled == C8@0:4
 				case "System.Boolean":
+				// CoreNFC.NFCTypeNameFormat enum is byte
+				case "CoreNFC.NFCTypeNameFormat":
```

Comment:
What was the error here ?
Afaik this convertion should be handled by the tests, it's not the first byte based enum